lib/controllers/project.js

```diff
@@ -761,8 +760,31 @@ Project.prototype.detectProjectType = function (path, callback) {
   */
 //-----------------------------------------------------------------------------
 Project.prototype.isSupported = function (srcVersion, runtime) {
+  var imageVersion, majorPass, minorPass, patchPass;
```

Comment:
This might be risky because of the way Meteor does versioning.. it's _not_ semver. :/ I tried doing this in Demeteorizer for a while and had to make changes to the parsing logic to use semver every other Meteor release or so.